.circleci/config.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:latest
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+          - dependencies-{{ checksum "package.json" }}
```

Comment:
I wonder if we should `checksum "yarn.lock"` instead? Seems like that may be better?

Reply:
I agree!